CRM/Contribute/Form/Contribution/Main.php

```diff
@@ -752,15 +741,35 @@ public static function buildRecur(&$form) {
 
     $unitVals = explode(CRM_Core_DAO::VALUE_SEPARATOR, $frUnits);
 
+    // FIXME: Ideally we should freeze select box if there is only
+    // one option but looks there is some problem /w QF freeze.
+    //if ( count( $units ) == 1 ) {
+    //$frequencyUnit->freeze( );
+    //}
+
+    $form->add('text', 'installments', ts('installments'),
+      $attributes['installments']
+    );
+    $form->addRule('installments', ts('Number of installments must be a whole number.'), 'integer');
+
+    $is_recur_label = ts('I want to contribute this amount every');
+    $unit_label = ts('Every');
```

Comment:
Where does this get used?

Reply:
Good call, it does not get used and I've now removed it.